ecs/elections/models.py

```diff
@@ -27,6 +27,13 @@ def is_set_up(self):
         else:
             return False
 
+    def is_generated(self):
+        """
+        Indicates if elections where
+        generated from normal distribution.
+        """
+        return self.candidates.values_list('position__x', 'position__y')[0][0] is not None
```

Comment:
a jak nie będzie żadnego kandydata jeszcze i ktoś wywoła to się wysypie ;)

coś w stylu
```try:
   if self.candidate.all()[0].position:
       return True
    else return false
    except return false
```